app/helpers/compute_resources_vms_helper.rb

```diff
@@ -237,13 +237,23 @@ def vm_host_action(vm)
 
   def vm_import_action(vm, html_options = {})
     return unless Host.for_vm(@compute_resource, vm).empty?
-    display_link_if_authorized(
-      _("Import"),
+    link1 = display_link_if_authorized(
```

Comment:
Nit: link1 is very unspecific, can we either name this `import_as_managed_link` or push to an array and use `safe_join`?